interface-definitions/interfaces-pppoe.xml.in

```diff
@@ -50,6 +50,20 @@
               <constraintErrorMessage>Host-uniq must be specified as hex-adecimal byte-string (even number of HEX characters)</constraintErrorMessage>
             </properties>
           </leafNode>
+          <leafNode name="holdoff">
+            <properties>
+              <help>Delay before re-dial to the access concentrator when PPP session terminated by peer (in seconds)</help>
+              <valueHelp>
+                <format>u32:1-86400</format>
+                <description>Holdoff time in seconds</description>
```

Comment:
From some examples, you might want to set “holdoff 0” to reconnect without waiting

Reply:
Okay